docs/get-started/video.md

```diff
@@ -75,7 +75,7 @@ Cosmos-Embed1 is generally better than InternVideo2 for most video embedding tas
 
 :::
```

Comment:
```{note}
Cosmos-Embed1 is generally better than InternVideo2 for most video embedding tasks. Consider using Cosmos-Embed1 (`cosmos-embed1-224p`) unless you have specific requirements for InternVideo2.
```

This should either be outside "PyPi With Internvideo2" or have it duplicate for "Source with internvideo2" as well